deploy/production/ingress.yml

```diff
@@ -40,3 +40,59 @@ spec:
             name: nginx-service
             port:
               number: 8080
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: justice-gov-uk-production-ingress-modsec
+  namespace: justice-gov-uk-production
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: justice-gov-uk-production-ingress-modsec-justice-gov-uk-production-green
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/enable-modsecurity: "true"
+    nginx.ingress.kubernetes.io/modsecurity-snippet: |
+      # SecAuditEngine On
+      # SecRuleEngine DetectionOnly
```

Comment:
@EarthlingDavey are these supposed to be commented out/can they be removed if we're not using?

Reply:
Good shout, I've removed them.